integration/e2e/limits_test.go

```diff
@@ -239,6 +239,9 @@ func TestQueryLimits(t *testing.T) {
 
 	require.Eventually(t, func() bool {
 		_, err = client.QueryTrace(tempoUtil.TraceIDToHexString(traceID[:]))
+		if err == nil {
+			return false
+		}
```

Comment:
Is this correct? The trace might not be immediately too big because it's sent in batches.